src/components/Dashboard/Header/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { ReactComponent as NimbleLogoWhite } from 'assets/images/icons/nimble-logo-white.svg';
+
+interface DashboardHeaderProps {
+  dateTime: string;
+  daysAgo: string;
+  profileUrl: string;
+  children: React.ReactNode;
+  'data-test-id'?: string;
+}
+const DashboardHeader = ({ dateTime, daysAgo, profileUrl, children, ...attributes }: DashboardHeaderProps): JSX.Element => {
+  return (
+    <div className="flex flex-col min-h-screen" {...attributes}>
```

Comment:
We should use the [semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp)
So please note that to use later 🙏

Reply:
Thanks a lot 😍